responsibleai/responsibleai/_internal/_forecasting_wrappers.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+# All wrappers in this file will be migrated to ml-wrappers.
+
+import inspect
+import sys
+
+import pandas as pd
+from ml_wrappers.model.base_wrapped_model import BaseWrappedModel
+
+from responsibleai.rai_insights.constants import ModelTask
+
+from .constants import _Forecasting
+
+_AZUREML = "azureml"
+_SKTIME = "sktime"
+
+
+def _get_model_package(model):
+    module = inspect.getmodule(model)
+    module_name = module.__name__.partition('.')[0]
+    model_package = sys.modules[module_name].__package__
+    return model_package
+
+
+def _wrap_model(model, examples, time_feature, time_series_id_features):
+    """Choose the appropriate wrapper for the model.
+
+    If the model has a forecast_quantiles method
+    (or in the sktime case: predict_quantiles) then choose the wrapped
+    quantile forecasting model, otherwise the basic wrapper without
+    quantile forecasting capabilities.
+    """
+    model_package = _get_model_package(model)
+    if ((model_package == _SKTIME and hasattr(model, "predict_quantiles")) or
+            hasattr(model, _Forecasting.FORECAST_QUANTILES)):
+        return _WrappedQuantileForecastingModel(
+            model, examples, time_feature, time_series_id_features)
+    else:
+        return _WrappedForecastingModel(
+            model, examples, time_feature, time_series_id_features)
+
+
+class _WrappedForecastingModel(BaseWrappedModel):
+    def __init__(self, model, examples, time_feature,

```

Comment:
I think all of these methods should have docs on them eventually.. I guess maybe it's fine for now since this is private and probably temporary anyway before it's moved to ml-wrappers but the docs should be added at some point

Reply:
I agree 100%
I'll add docs in this PR.

Reply:
Done.